packages/hurl/src/http/headers_helper.rs

```diff
@@ -124,7 +124,12 @@ mod tests {
         let mut headers = HeaderVec::new();
         headers.push(Header::new("Host", "localhost:8000"));
```

Comment:
What can be good is to have also a "Repeated-Header" in headers, prior to do the aggregation of the two headers lists: this way we test also what's happens where there is the same header name in the two headers lists.

Reply:
@jcamiel Done 👍